Formula/hazelcast-member.rb

```diff
@@ -0,0 +1,46 @@
+class HazelcastMember < Formula
+  desc "Command-line tool for running Hazelcast member instance(s)"
+  homepage "https://github.com/hazelcast/hazelcast"
+  url "https://github.com/hazelcast/hazelcast-member-tool/releases/download/v3.11.0/hazelcast-member-3.11.0.tar.gz"
+  sha256 "13d9d8f9c608ea8f672e58dba87500ff38c2c70d1f69e1f3cf30532a9be67056"
+
+  bottle :unneeded
+
+  depends_on "coreutils"
+  depends_on :java => "1.7+"
+
+  def install
+    mkpath "#{etc}/hazelcast"
```

Comment:
Not sure the `mkpath` is necessary, I think it will be created by `install_symlink`